ext/puma_http11/puma_http11.c

```diff
@@ -181,7 +181,11 @@ void http_field(puma_parser* hp, const char *field, size_t flen,
     f = rb_str_new(hp->buf, new_size);
   }
 
-  while (vlen > 0 && isspace(value[vlen - 1])) vlen--;
+  while (vlen > 0 && (value[vlen - 1] == ' ' || value[vlen - 1] == '\t')) vlen--;
```

Comment:
(I think, C is not my daily driver) We should be using `isspace` instead of rolling our own check. The `isspace` call already handles tab:

- https://man.openbsd.org/isspace.3
- https://manpages.debian.org/testing/manpages-dev/isspace.3.en.html

The code on main already strips leading whitespace, the reason your test fails on main is because of the trailing whitespace.

Reply:
`isspace` strips more than *just* space and tab, though. It also strips `\x0b` and `\x0c`, for example. It's also dependent on locale. This is why I chose to compare to space and tab.

Reply:
Would be worth having a `is_ows(const char c)` for readability though.

Other than that the C code looks fine.

Reply:
Agreed, and now done.